### PR TITLE
Website: Fabric to fluent 3

### PR DESCRIPTION
**Pull request by micahgodbolt**
Focus of this PR is to address all instances of  `Fabric` (uppercase) and `fabric` (lowercase) in the docs.  

This PR is not meant to capture EVERY instance of the word "fabric". PR review should be focused on the changed text only.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12334)

### CODE REVIEW

File: apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedNextSteps.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The [Fluent UI React wiki](https://github.com/microsoft/fluentui/wiki) has infor
 
 Fluent UI React components are highly customizable using CSS-in-JS. We also support customization of colors and more throughout the entire app using themes.
 
-For an overview of styling and theming approaches in Fabric, see [this page](https://github.com/Microsoft/frontend-bootcamp/tree/master/step2-03/demo). Then check out the links below for more details:
+For an overview of styling and theming approaches in Fluent UI, see [this page](https://github.com/Microsoft/frontend-bootcamp/tree/master/step2-03/demo). Then check out the links below for more details:
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedNextSteps.md]: We should ideally also update the frontend-bootcamp code to use the new package (at least add a work item for that somewhere?)

File: apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedOverview.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 Fluent UI for the web is available in two flavors: Fluent UI React and Fabric Core.
 
 <!-- manually creating h3 to avoid duplicate auto-generated IDs -->
-<h3 class="fabric-react-overview">Fluent UI React</h3>
+<h3 class="fluent-ui-react-overview">Fluent UI React</h3>
 
 Fluent UI React is the official [open-source](https://github.com/microsoft/fluentui) React front-end framework designed to build experiences that fit seamlessly into a broad range of Microsoft products. It provides robust, up-to-date, accessible components which are highly customizable using CSS-in-JS. [Get started with Fluent UI React](#fabric-react)
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedOverview.md]: Just to make sure this doesn't get missed somehow--this is auto-generated based on the title of the section and therefore should be updated.
```suggestion
Fluent UI React is the official [open-source](https://github.com/microsoft/fluentui) React front-end framework designed to build experiences that fit seamlessly into a broad range of Microsoft products. It provides robust, up-to-date, accessible components which are highly customizable using CSS-in-JS. [Get started with Fluent UI React](#fluent-ui-react)
```

File: apps/fabric-website/src/pages/Styles/Colors/docs/web/ColorsImplementation.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ### Fluent UI React (JavaScript variables)
 
 ```jsx
-import { CommunicationColors } from '@uifabric/fluent-theme/lib/fluent/FluentColors';
+import { CommunicationColors } from '@fluentui/fluent-theme/lib/fluent/FluentColors';
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/Colors/docs/web/ColorsImplementation.md]: Probably don't change this yet--we should do an all-up pass for `@uifabric => @fluentui` renames **if/when** we decide to make that change, which I'm not sure if we have yet.

File: apps/fabric-website/src/pages/Styles/ElevationPage/docs/web/ElevationImplementation.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ### Fluent UI React (JavaScript variables)
 
 ```jsx
-import { Depths } from '@uifabric/fluent-theme/lib/fluent/FluentDepths';
+import { Depths } from '@fluentui/fluent-theme/lib/fluent/FluentDepths';
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/ElevationPage/docs/web/ElevationImplementation.md]: Same here

File: apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
-The most durable and future-proof way to get the right icon for a file or item is to use the [file-type-icons package](https://github.com/microsoft/fluentui/tree/master/packages/file-type-icons) from Fluent UI (`@uifabric/file-type-icons` on npm).
+The most durable and future-proof way to get the right icon for a file or item is to use the [file-type-icons package](https://github.com/microsoft/fluentui/tree/master/packages/file-type-icons) from Fluent UI (`@fluentui/file-type-icons` on npm).
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md]: Same here about scope names (+2 later in this file)

File: apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
 
-The following code shows you how to specify a file type icon by extension, item type, icon size, and image type using the `file-type-icons` package along with Fabric's `<Icon>` component:
+The following code shows you how to specify a file type icon by extension, item type, icon size, and image type using the `file-type-icons` package along with Fluent UI's `<Icon>` component:
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md]: While we're here
```suggestion
The following code shows you how to specify a file type icon by extension, item type, icon size, and image type using the `@uifabric/file-type-icons` package along with Fluent UI's `<Icon>` component:
```

File: apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md [hunk 7]
```diff
@@ -19,7 +19,7 @@ When specifying `size`, stick to these default sizes so the images appear as int
...
 
-Here's a [simple demo](https://github.com/microsoft/fluentui/blob/master/packages/experiments/src/components/FileTypeIcon/examples/FileTypeIcon.Basic.Example.tsx) of how to use the [file-type-icons package](https://github.com/microsoft/fluentui/tree/master/packages/file-type-icons).  The code for the main method you'll use ([`getFileTypeIconProps`](https://github.com/microsoft/fluentui/blob/master/packages/file-type-icons/src/getFileTypeIconProps.ts)) is also well-commented. The icons are kept up-to-date on Fabric's CDN, so we recommend not copying or reference these assets directly.
+Here's a [simple demo](https://github.com/microsoft/fluentui/blob/master/packages/experiments/src/components/FileTypeIcon/examples/FileTypeIcon.Basic.Example.tsx) of how to use the [file-type-icons package](https://github.com/microsoft/fluentui/tree/master/packages/file-type-icons).  The code for the main method you'll use ([`getFileTypeIconProps`](https://github.com/microsoft/fluentui/blob/master/packages/file-type-icons/src/getFileTypeIconProps.ts)) is also well-commented. The icons are kept up-to-date on Fluent UI's CDN, so we recommend not copying or reference these assets directly.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md]: Should probably add the scope to the file-type-icons name here too
```suggestion
Here's a [simple demo](https://github.com/microsoft/fluentui/blob/master/packages/experiments/src/components/FileTypeIcon/examples/FileTypeIcon.Basic.Example.tsx) of how to use the [`@uifabric/file-type-icons` package](https://github.com/microsoft/fluentui/tree/master/packages/file-type-icons).  The code for the main method you'll use ([`getFileTypeIconProps`](https://github.com/microsoft/fluentui/blob/master/packages/file-type-icons/src/getFileTypeIconProps.ts)) is also well-commented. The icons are kept up-to-date on Fluent UI's CDN, so we recommend not copying or reference these assets directly.
```

File: apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyImplementation.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ### Fluent UI React (JavaScript variables)
 
 ```tsx
-import { FontSizes } from '@uifabric/fluent-theme/lib/fluent/FluentType';
+import { FontSizes } from '@fluentui/fluent-theme/lib/fluent/FluentType';
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyImplementation.md]: same about scope

**Comment by ecraig12345** [comment]
One other small thing I think you should go ahead and update (so it doesn't get lost) is this link in `IconOverview.md`:
`[icon documentation](#/styles/web/icons#fabric-react)`
which should now be this (due to the section title change in FabricIconsPage):
`[icon documentation](#/styles/web/icons#fluent-ui-react)`

File: apps/fabric-website/homepage.htm [hunk 9]
```diff
@@ -1,7 +1,7 @@
 <div
   id="metadata"
   title="Home - Fluent UI"
-  description="The official front-end framework for building experiences that fit seamlessly into Office and Office 365."
+  description="The official front-end framework for building experiences that fit seamlessly into Microsoft 365."
```
Comment (ecraig12345) [inline apps/fabric-website/homepage.htm]: We probably need to do a larger pass throughout the repo to fix lotttsssss more instances, but this is a good start. 👍

File: apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedNextSteps.md (hunk 1 above)
Comment (micahgodbolt) [reply apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedNextSteps.md]: https://github.com/microsoft/frontend-bootcamp/issues/179

File: apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedOverview.md (hunk 2 above)
Comment (micahgodbolt) [reply apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedOverview.md]: ah...i changed some of those...missed that one it seems

File: apps/fabric-website/src/pages/Styles/ElevationPage/docs/web/ElevationImplementation.md (hunk 4 above)
Comment (micahgodbolt) [reply apps/fabric-website/src/pages/Styles/ElevationPage/docs/web/ElevationImplementation.md]: weird...those slipped through. i know i changed similar ones elsewhere. my search foo was failing

File: apps/fabric-website/homepage.htm (hunk 9 above)
Comment (micahgodbolt) [reply apps/fabric-website/homepage.htm]: yeah...website copy is clean of incorrect 'office' stuff

**Pull request merged**